microceph/client/remote.go

```diff
@@ -0,0 +1,62 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/canonical/lxd/shared/api"
+	"github.com/canonical/lxd/shared/logger"
+	"github.com/canonical/microceph/microceph/api/types"
+	microCli "github.com/canonical/microcluster/client"
+	"github.com/canonical/microcluster/state"
+)
+
+// Sends the desired list of services to be restarted on every other member of the cluster.
+func SendRemoteImportRequest(ctx context.Context, c *microCli.Client, data types.Remote) error {
+	queryCtx, cancel := context.WithTimeout(ctx, time.Second*120)
+	defer cancel()
+
+	err := c.Query(queryCtx, "PUT", types.ExtendedPathPrefix, api.NewURL().Path("client", "remotes", data.Name), data, nil)
+	if err != nil {
+		return fmt.Errorf("failed to import MicroCeph remote: %w", err)
+	}
+
+	return nil
+}
+
+// Sends the remote import request to every other member of the cluster.
```

Comment:
Nit: func doc convention:

// SendRemoteImportToClusterMembers sends the ...

Reply:
ack